src/backends/sentry_backend_crashpad.cpp

```diff
@@ -439,9 +439,13 @@ crashpad_backend_startup(
     if (minidump_url) {
         SENTRY_DEBUGF("using minidump URL \"%s\"", minidump_url);
     }
+    const char *proxy_url;
+#ifdef SENTRY_PLATFORM_MACOS
+    proxy_url = getenv("https_proxy"); // other platforms do this already
+#endif
+    proxy_url = options->proxy ? options->proxy : proxy_url ? proxy_url : "";
```

Comment:
Currently we only look at `https_proxy`, but maybe if this is empty we should check `http_proxy` as well? (although these two should ideally hold the same value). 
Afaict cURL doesn't use `http_proxy` when trying to send to a configured `https://` DSNs, so I would keep our implementation as similar as possible.

Reply:
Yeah, we talked about this once. I think a "no fallback" if `DSN` has a `https`-scheme use `https_proxy`; otherwise, use `http_proxy` is enough.